packages/common/src/configuration/runtime-config/runtime-config.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "commonConfig": {
```

Comment:
>  [](start = 0, length = 1)

Should we move this file under \packages\config ?

Reply:
i didnt want to create a package for every small feature that we have. we can move this later to config package based on complexity.